deterministic.c

```diff
@@ -85,7 +90,7 @@ int falcon_det1024_convert_compressed_to_ct(void *sig_ct,
 	}
 
         // Decode signature's s_bytes into 1024 signed-integer coefficients.
-	v = Zf(comp_decode)(coeffs, FALCON_DET1024_LOGN, sig_compressed+2, sig_compressed_len-2);
+	v = Zf(comp_decode)(coeffs, FALCON_DET1024_LOGN, ((uint8_t*)sig_compressed)+2, sig_compressed_len-2);
```

Comment:
this cast is because the size of a void* can be 4 or 8 bytes, depending on platform right — but you are only doing it for the +2 case, not the -2 case?

Reply:
Actually the size of void* is not well defined. and it is not recommended to add or subtract to this pointer.
if you want to count in bytes, need to cast it into a byte pointer == uint8

Reply:
right, my point is I think you also need to do
```suggestion
	v = Zf(comp_decode)(coeffs, FALCON_DET1024_LOGN, ((uint8_t*)sig_compressed)+2, (uint8_t*)sig_compressed_len-2);
```

Reply:
yes, right, OK, i thought it said sig_compressed-2 at the end there.